### PR TITLE
Update the deploy script

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add a compound governor deploy
- Add a regular governor deploy

Closes https://github.com/ScopeLift/l2-flexible-voting-frontend/issues/12

### CODE REVIEW

File: script/helpers/Governors.sol [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+import {TimelockController} from "openzeppelin-flexible-voting/governance/TimelockController.sol";
+import {GovernorTimelockCompound} from
+  "openzeppelin-flexible-voting/governance/extensions/GovernorTimelockCompound.sol";
+import {ICompoundTimelock} from "openzeppelin-flexible-voting/vendor/compound/ICompoundTimelock.sol";
+
+import {GovernorCountingFractional} from "flexible-voting/src/GovernorCountingFractional.sol";
+
+contract GovernorTestnetSettings {
+  function quorum(uint256) public view virtual returns (uint256) {
+    return 1_000_000;
```
Comment (apbendi) [inline script/helpers/Governors.sol:26]: Shouldn't this be `1_000_000e18` if we want it to produce results that look like a "real" DAO with our Decimals 18 tokens?

File: script/helpers/Governors.sol [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+  function votingDelay() public view virtual returns (uint256) {
+    return 90;
+  }
+
+  function votingPeriod() public view virtual returns (uint256) {
+    return 1800;
+  }
+
+  function proposalThreshold() public view virtual returns (uint256) {
+    return 500_000;
```
Comment (apbendi) [inline script/helpers/Governors.sol:38]: Same as above—should be `e18`

File: src/L1VotePool.sol [hunk 3]
```diff
@@ -41,6 +42,7 @@ abstract contract L1VotePool {
   /// @param _governor The address of the L1 Governor contract.
   constructor(address _governor) {
     GOVERNOR = IGovernor(_governor);
+    console2.logAddress(address(GOVERNOR));
```
Comment (apbendi) [inline src/L1VotePool.sol:45]: leftover from debugging and needs to be removed?

File: src/FakeERC20.sol [hunk 4]
```diff
@@ -2,6 +2,7 @@
 pragma solidity ^0.8.20;
 
 import {ERC20Votes} from "openzeppelin/token/ERC20/extensions/ERC20Votes.sol";
+
```
Comment (apbendi) [inline src/FakeERC20.sol:5]: nit: can undo this change to avoid unneccessary diff on this file

File: script/WormholeL2FlexibleVotingDeploy.s.sol [hunk 5]
```diff
@@ -115,4 +137,46 @@ contract WormholeL2FlexibleVotingDeploy is Script, ScriptConstants {
     vm.broadcast();
     l1TokenBridge.initialize(address(l2Token));
   }
+
+  function _setupGovernor() internal returns (address, address, bool) {
```
Comment (apbendi) [inline script/WormholeL2FlexibleVotingDeploy.s.sol:141]: maybe a quick comment and/or named returned vars to help make it clear what this function returns?

File: script/WormholeL2FlexibleVotingDeploy.s.sol [hunk 6]
```diff
@@ -115,4 +137,46 @@ contract WormholeL2FlexibleVotingDeploy is Script, ScriptConstants {
...
+    address governorAddress = vm.envOr("L1_GOVERNOR_ADDRESS", address(0));
+    address l1TokenAddress = vm.envOr("L1_TOKEN_ADDRESS", address(0));
+    bool compToken = vm.envOr("L1_COMP_TOKEN", false);
```
Comment (apbendi) [inline script/WormholeL2FlexibleVotingDeploy.s.sol:144]: nit: prefer `isCompToken` naming for bools

**Review by apbendi — COMMENTED** [review]
Thanks @alexkeating left a few comments

File: script/helpers/Governors.sol (hunk 1 above)
Comment (alexkeating) [reply script/helpers/Governors.sol]: https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea

File: script/helpers/Governors.sol (hunk 2 above)
Comment (alexkeating) [reply script/helpers/Governors.sol]: https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea

File: src/L1VotePool.sol (hunk 3 above)
Comment (alexkeating) [reply src/L1VotePool.sol]: https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea

File: script/WormholeL2FlexibleVotingDeploy.s.sol (hunk 6 above)
Comment (alexkeating) [reply script/WormholeL2FlexibleVotingDeploy.s.sol]: https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea

File: src/FakeERC20.sol (hunk 4 above)
Comment (alexkeating) [reply src/FakeERC20.sol]: https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/c4ce2317d5e8abeea931f92253011b96e68d0bda

**Review by apbendi — APPROVED** [review]
LGTM thanks @alexkeating
